src/index.html

```diff
@@ -2,16 +2,174 @@
+          id="oldAreYou"
+          value="12"
+          class="oldAreYou"
+          >
+        </label>
+        <label for="dateOfBIrth" class="margin10Px">
+          Full date of birth:
+          <input
+          type="date"
+          id="dateOfBIrth"
+          >
+        </label>
+        <label for="acceptAgreement" class="margin10Px lastMargin">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="checkboxAccept"
+          id="acceptAgreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="displayInput">
+        <legend>Registration</legend>
+        <label
+        for="email"
+        >
+          E-mail:
+          <input
+          type="email"
+          id="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label
+        for="password"
+        class="margin10Px"
+        >
+          Password:
+          <input
+          type="password"
+          id="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="displayInput">
+        <legend>An interesting fact about you!</legend>
+        <label>
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            value="Yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            value="No"
+            >
+            No
+          </label>
+        </label>
+        <label for="choseColor">
+          What is your favorite color?
+          <input
+          type="color"
+          id="choseColor"
+          class="margin10Px"
+          >
+        </label>
+        <label for="chooseTime">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          id="chooseTime"
+          class="margin10Px"
+          >
+        </label>
+        <label for="chooseCar">
+          What are your favorite brand of cars?
+          <select
+          name="brandCar"
+          id="chooseCar"
+          class="margin10Px"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label for="rateWork">
+          How do you rate our work?
+          <input
+          type="range"
+          name="rateWork"
+          id="rateWork"
+          value="0"
+          class="margin8pxInInteresting"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="displayInput additionalInfo">
+        <legend>Additional info:</legend>
+        <label>
+          <label class="textarea">
+            Comments:
+            <textarea
+            name="textarea"
+            cols="20"
+            rows="2"
+            >
+            </textarea>
```

Comment:
```suggestion
            <textarea
            name="textarea"
            cols="20"
            rows="2"
            ></textarea>
```
![image](https://user-images.githubusercontent.com/97547824/201465395-0be7e21e-f4e5-4e6f-8d0b-023aa2f0c9b1.png)